src/styles/header.scss

```diff
@@ -0,0 +1,81 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
```

Comment:
It seems there's an import statement for a URL that is incomplete or missing. Please ensure that the URL is correct and that the resource being imported is necessary for the header styles.